language/en/types/extension.php

```diff
@@ -121,7 +121,7 @@
 [*]IRC Support - [url=irc://irc.freenode.net/phpBB-coding]#phpBB-coding[/url] is registered on the FreeNode IRC network ([url=irc://irc.freenode.net/]irc.freenode.net[/url])[/list]
 
 [b]If you wish to discuss anything in this PM please use the “Validation Discussion“ sticky topic located in your extension’s Discussion/Support tab.[/b]
-
+c
```

Comment:
What's this? `c`

Reply:
Typo :rofl: 
Fixed @ 8fb5aec912a6dacb7db05a20dd590ae5253f2cc1